solidity/deploy/26_authorize_maintainer_proxy_in_reimbursement_pool.ts

```diff
@@ -20,3 +20,10 @@ export default func
 
 func.tags = ["AuthorizeMaintainerProxyInReimbursementPool"]
 func.dependencies = ["ReimbursementPool", "MaintainerProxy"]
+
+// On mainnet, the ReimbursementPool ownership is passed to the  Treshold
+// Council / DAO and that address is not controlled by the dev team.
+// Hence, this step can be executed only for non-mainnet networks such as
+// Hardhat (unit tests) and Goerli (testnet).
+func.skip = async (hre: HardhatRuntimeEnvironment): Promise<boolean> =>
+  hre.network.name === "mainnet"
```

Comment:
Regarding #463, I think we should also add a point that the `ReimbursementPool` must be funded with Ethers. Otherwise, all transactions made through the `MaintainerProxy` that involve the refund will revert. That's rather obvious but may be good to write it down explicitly.

Reply:
I believe you meant #438? If so, we already have this step in the deployment scenario, see "Fund Reimbursement Pool" section.

Reply:
Oh yeah, right.